interestingness/diffTest.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+'''
+This file came from nbp's GitHub PR #2 for adding new Lithium reduction strategies.
+  https://github.com/MozillaSecurity/lithium/pull/2
```

Comment:
I think the authorship is interesting for blaming ( :) ), but less interesting for documentation strings.
Maybe move this to a `#` comment, and add the following `'''` comment:

```python
'''
This test is made to minimize a test case by comparing a single binary with different command line arguments.
This can be used to isolate and minimize differential behaviour test cases.
'''
```